keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -369,7 +388,11 @@ Based on reviewers feedback describe what additional tests need to be added prio
 implementing this enhancement to ensure the enhancements have also solid foundations.
 -->
 
-Aside from standard testing we will also be skew testing.
+We're planning to expand tests adding:
```

Comment:
all tests as a prerequisite for beta?

Reply:
https://github.com/kubernetes/kubernetes/pull/131619#pullrequestreview-2836473117

Reply:
So if we add `- config API fuzzy tests` and `input validation and correctness` in here, this will be requirement for beta promotion.

Reply:
That is correct, and that's why https://github.com/kubernetes/kubernetes/pull/131818 contains all the necessary tests. Although, more will likely followup.